azext_edge/edge/providers/stats.py

```diff
@@ -43,9 +43,14 @@ def _preprocess_stats(
         raise ResourceNotFoundError(
             f"Diagnostics service pod '{diag_service_pod_prefix}' does not exist in namespace '{namespace}'."
         )
-    diagnostic_pod = target_pods[0]
-
-    return namespace, diagnostic_pod
+    for pod in target_pods:
+        if pod.status.phase.lower() == PodState.running.name:
```

Comment:
can't we just leave it as the phase == PodState.running.value rather than lowercasing both?

or can pod phases be all lowercase?

Reply:
Ok, let's try the other way